go.mod

```diff
@@ -394,6 +394,10 @@ replace github.com/eclipse/paho.mqtt.golang => github.com/shivamkm07/paho.mqtt.g
 // Don't commit with this uncommented!
 //
 // replace github.com/dapr/components-contrib => ../components-contrib
+
 //
 // Then, run `make modtidy` in this repository.
 // This ensures that go.mod and go.sum are up-to-date.
+
+// This is the feature/pubsub-batching branch in components-contrib, will be removed before merging into master branch of dapr
+replace github.com/dapr/components-contrib => github.com/dapr/components-contrib v1.8.0-rc.1.0.20220920200252-e9e750bce615
```

Comment:
Is it planned to be merged this week? I do not want to introduce replace here.

Reply:
For sure ... it will be resolved before a PR is created to master ...